build-storybook.log

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Should we be committing this file? The file seem to be a build log, or is it needed by chromatic in some way?

Reply:
ah ya probably not, deleted it and added to the git ignore.

Reply:
great thanks!